tests/Feature/Thread/CreateThreadTest.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace Tests\Feature\Thread;
+
+use App\Models\Thread;
+use App\Models\User;
+use Illuminate\Foundation\Testing\DatabaseMigrations;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Illuminate\Foundation\Testing\WithFaker;
+use Tests\TestCase;
+
+class CreateThreadTest extends TestCase
+{
+    use DatabaseMigrations;
+
+    
+    public function testThreadCreateURL()
+    {
+        $this->actingAs(User::factory()->create());
+        $response = $this->get('threads/create');
+        $response->assertStatus(200);
+    }
+
+    public function testAuthenticatedUsersCanCreateNewThread()
+    {
+        //creates an user and authenticate
+        $this->actingAs(User::factory()->create());
```

Comment:
Blaze has a easy helper method for authenticating users. So `$this->actingAs();` can now be `$this->signIn();` and you don't have to pass in a user.